CE07SHSM/CE07SHSM_D00018_ingest.csv

```diff
@@ -0,0 +1,37 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+# mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/cpm3/syslog/cpm_status.txt,CE07SHSM-MFC31-00-CPMENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl36/syslog/*.syslog.log,CE07SHSM-MFD35-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.vel3d_cd.dcl.vel3d_cd_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl36/vel3d*/*.vel3d.log,CE07SHSM-MFD35-01-VEL3DD000,telemetered,Available,
```

Comment:
same here as above, the X directory shows "vel3d_decimated", just checking it won't be named that way when deployed.
I also see the optaa_decimated under dcl37.

Reply:
Same as for the optaa noted above